src/KingSystem/Map/mapObjectLink.cpp

```diff
@@ -19,4 +135,117 @@ bool ObjectLink::getObjectProcWithAccessor(act::ActorLinkConstDataAccess& access
         return other_obj->getActorWithAccessor(accessor);
 }
 
+ObjectLinkData::ObjectLinkData() = default;
+
+void ObjectLinkData::deleteArrays() {
+    if (mRails) {
+        delete mRails;
+        mRails = nullptr;
+    }
+
+    if (mLinksOther.links) {
```

Comment:
I suspect this is actually a sead::Buffer::freeBuffer (four calls to freeBuffer for different link arrays).